packer/install-dependencies.sh

```diff
@@ -32,145 +64,177 @@ for VNET_CNI_VERSION in $VNET_CNI_VERSIONS; do
     VNET_CNI_PLUGINS_URL="https://acs-mirror.azureedge.net/cni/azure-vnet-cni-linux-amd64-v${VNET_CNI_VERSION}.tgz"
     downloadAzureCNI
 done
+echo "  - Azure CNI versions: ${VNET_CNI_VERSIONS}" >> /var/log/azure/golden-image-install.complete
 
 for CNI_PLUGIN_VERSION in $CNI_PLUGIN_VERSIONS; do
     CNI_PLUGINS_URL="https://acs-mirror.azureedge.net/cni/cni-plugins-amd64-v${CNI_PLUGIN_VERSION}.tgz"
     downloadCNI
 done
+echo "  - CNI plugin versions: ${CNI_PLUGIN_VERSIONS}" >> /var/log/azure/golden-image-install.complete
 
 CONTAINERD_DOWNLOAD_URL_BASE="https://storage.googleapis.com/cri-containerd-release/"
 for CONTAINERD_VERSION in ${CONTAINERD_VERSIONS}; do
     downloadContainerd
 done
+echo "  - containerd versions: ${CONTAINERD_VERSIONS}" >> /var/log/azure/golden-image-install.complete
 
 installImg
+echo "  - img" >> /var/log/azure/golden-image-install.complete
+
+echo "Docker images pre-pulled:" >> /var/log/azure/golden-image-install.complete
 
 DASHBOARD_VERSIONS="1.10.1"
 for DASHBOARD_VERSION in ${DASHBOARD_VERSIONS}; do
     pullContainerImage "docker" "k8s.gcr.io/kubernetes-dashboard-amd64:v${DASHBOARD_VERSION}"
 done
+echo "  - k8s.gcr.io/kubernetes-dashboard-amd64 ${DASHBOARD_VERSIONS}" >> /var/log/azure/golden-image-install.complete
```

Comment:
Slight preference for just moving these into the for loop and printing each image in its own line with the full "<image>:<version>" string

Reply:
Actually, same comment for all component + version installs above and below

Reply:
yeah makes sense

Reply:
done